modules/bmo-performing-a-live-update-to-the-hostfirmwarecomponents-resource.adoc

```diff
@@ -2,22 +2,32 @@
 //
 // * installing/installing_bare_metal/ipi/ipi-install-post-installation-configuration.adoc
 :_mod-docs-content-type: PROCEDURE
-[id="bmo-patching-the-hostfirmwarecomponents-resource_{context}"]
-= Patching the HostFirmwareComponents resource
+[id="bmo-performing-a-live-update-to-the-hostfirmwarecomponents-resource_{context}"]
+= Performing a live update to the HostFirmwareComponents resource
 
-To patch the `HostFirmwareComponents` resource for a provisioned host, perform the following:
+You can perform a live update to the `HostFirmareComponents` resource after it has begun running workloads. Live updates do not detach the host from Ironic and reattach it after performing the update. To update the `HostFirmwareComponents` resource, perform the following actions:
```

Comment:
suggestion: Live updates can be performed on an already provisioned node and do not trigger deprovisioning and re-provisioning of the node. To update the `HostFirmwareComponents` resource, perform the following actions: